### PR TITLE
Simplify Casting

### PR DESCRIPTION
**Pull request by axchu**
### Fixes
- Fixes https://github.com/ethereum-optimism/roadmap/issues/278

## Contributing Agreement

- [x] I have read and understood the [Optimism Contributing Guide and Code of Conduct](https://github.com/ethereum-optimism/optimism-monorepo/blob/master/.github/CONTRIBUTING.md) and am following those guidelines in this pull request.

### CODE REVIEW

**Comment by axchu** [comment]
note that abi.decode does not work for < 32 byte length

File: contracts/optimistic-ethereum/libraries/utils/Lib_BytesUtils.sol [hunk 1]
```diff
@@ -177,11 +177,15 @@ library Lib_BytesUtils {
...
     {
-        bytes32 ret;
-        assembly {
-            ret := mload(add(_bytes, 32))
+        if (_bytes.length < 32) {
+            bytes32 ret;
+            assembly {
+                ret := mload(add(_bytes, 32))
+            }
+           return ret;
```
Comment (kfichter) [inline contracts/optimistic-ethereum/libraries/utils/Lib_BytesUtils.sol:185]: Indent is wrong here.

**Review by kfichter — APPROVED** [review]
Requesting minor indent fix, otherwise LGTM.

**Comment by kfichter** [comment]
Sorry meant to request changes -- could you just make sure to fix the indent before merge?

**Pull request merged**